api/api.yaml

```diff
@@ -1162,6 +1162,120 @@ paths:
               schema:
                 $ref: "../problem/problem.yaml#/components/schemas/ApiProblem"
       x-codegen-request-body-name: installationUpdate
+  /projects/{projectIdOrName}/integrations/{integrationId}/installations/{installationId}:pause:
+    post:
+      summary: Pause reads for an installation
+      description: Pauses all reads for an installation. If reads are already paused, this will be a no-op. Accepts specific objects to pause reads for in the request body.
+      operationId: pauseReads
+      tags: ["Installation"]
+      parameters:
+        - name: projectIdOrName
+          in: path
+          required: true
+          description: The Ampersand project ID or project name.
+          schema:
+            type: string
+          example: my-project
+        - name: integrationId
+          in: path
+          required: true
+          description: The integration ID.
+          schema:
+            type: string
+        - name: installationId
+          in: path
+          required: true
+          description: The Ampersand installation ID.
+          schema:
+            type: string
+      requestBody:
+        required: false
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                objects:
+                  type: array
```

Comment:
can you note that this is an optional parameter and if omitted we will pause all objects? you should also specify whether the builder can use mapped object names here. (I think if it's easy to support, that would be great, but if it's hard, then don't support it, but make it clear here.)